client/me/get-apps/apps-badge.jsx

```diff
@@ -73,6 +73,8 @@ export class AppsBadge extends PureComponent {
 			this.image = null;
 			this.loadImage();
 		}
+
+		this.onLinkClick = this.onLinkClick.bind( this );
```

Comment:
This is a real bug that causes exceptions when someone clicks on the link. A better fix would be to convert `onLinkClick` to a class property that has an arrow function as value.